jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/validation/BPMN2SyntaxChecker.java

```diff
@@ -96,15 +97,15 @@ public void checkSyntax() {
                             foundPackageName = true;
                             pname = (String) entry.getValue();
                             if(isEmpty(pname)) {
-                                addError(defaultResourceId, new ValidationSyntaxError(process, BPMN2_TYPE,  "Process has no package name."));
+                                addError(defaultResourceId, new ValidationSyntaxError(process, BPMN2_TYPE, SyntaxCheckerErrors.PROCESS_HAS_NO_PACKAGE_NAME));
                             }
                             if(!isValidPackageName(pname)) {
                                 addError(defaultResourceId, new ValidationSyntaxError(process, BPMN2_TYPE,  "Package name contains invalid characters."));
```

Comment:
Lots of syntax error constants have been moved into SyntaxCheckerErrors.java, but there are still quite a few constants in this class, such as here. It would be nice to move them into SyntaxCheckerErrors.